keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -1446,8 +1769,12 @@ For each of them, fill in the following information by copying the below templat
     - Testing: Are there any tests for failure mode? If not, describe why.
 -->
 
+None.
```

Comment:
That is not something I can easily believe.
Even if we didn't observe, there are definitely failure modes that we should mention..

@SergeyKanzhelev